gradio/strings.py

```diff
@@ -19,8 +19,7 @@
     "COLAB_DEBUG_TRUE": "Colab notebook detected. This cell will run indefinitely so that you can see errors and logs. "
     "To turn off, set debug=False in launch().",
     "COLAB_DEBUG_FALSE": "Colab notebook detected. To show errors in colab notebook, set debug=True in launch()",
-    "COLAB_BETA": "\nUsing Embedded Colab Mode (NEW). If you have issues, please use share=True and file an issue at https://github.com/gradio-app/gradio/"
-    "\nNote: opening the browser inspector may crash Embedded Colab Mode.",
+    "COLAB_WARNING": "Opening Chrome Inspector may crash Colab Embedded Mode.",
```

Comment:
Nit: is it clear what "Colab Embedded Mode" is? Perhaps better to rephrase:
```suggestion
    "COLAB_WARNING": "Note: opening Chrome Inspector may crash demo inside Colab notebooks.",
```